CoordImporter/Parser/Parsers.cs

```diff
@@ -101,10 +104,14 @@ public class BearParser : ITrackerParser
 {
     // For the format "Labyrinthos ( 16.5 , 16.8 ) Storsie"
     private static readonly Regex BearRegex = new Regex(
-        @"(?<map_name>[\D'\s+-]*)\s*(?<instance>[1-9]?)\s*\(\s*(?<x_coord>[0-9\.]+)\s*,\s*(?<y_coord>[0-9\.]+)\s*\)\s*(?<mark_name>[\w+ +-]+)",
```

Comment:
i actually noticed these extra `+`s in a lot of the character classes (`[...]`) of the regexes. can there actually be `+` characters in mark and map names and such? or are those `+`s there for another reason? :thinky:

Reply:
It's probably me not remembering how character classes work. IIRC having `\s+` was my way of saying "there might be multiple spaces here", which is dumb because that's not how character classes work. They can be removed because there are no pluses in any of the text fields that the importer cares about AFAIR.

Reply:
oh! that makes sense XD. i'll pull them out 👍